README.md

```diff
@@ -0,0 +1,3 @@
+# Kaibutsu
+
+Kernel Buildsystem for devices that need a custom kernel to run Ultramarine Linux.
```

Comment:
this is kinda clunky

Reply:
How about 'Buildsystem for making custom kernels for devices not yet ready for our base kernel.'

Reply:
ai sounding shit

```suggestion
Buildsystem for patched device kernels.
```